src/Admin.php

```diff
@@ -136,12 +136,22 @@ public static function post_action_label($post_id) {
       $discount_percentage = 0;
     }
 
+    $formatted_regular_price = wc_price($regular_price, ['price_format' => '%1$s&nbsp;%2$s']);
+    // Replace ",00" with ",-".
+    $formatted_regular_price = preg_replace('/\,0++/', ',-', $formatted_regular_price);
```

Comment:
I think you're looking for exactly "00", so you can avoid the complexity of PCRE:
```suggestion
    $formatted_regular_price = str_replace($formatted_regular_price, '@,00@', ',—');
```

Reply:
Unfortunately, I couldn't make it work with `str_replace`. Probably it has something todo with `$formatted_regular_price` being multiline HTML. I'm using `preg_replace('@,00@', ',–', $formatted_regular_price)` now.